src/renderer.js

```diff
@@ -209,7 +209,7 @@ class Renderer {
      * import tags to prevent further loading of resources.
      */
     function stripPage() {
-      const elements = document.querySelectorAll('script, link[rel=import]');
+      const elements = document.querySelectorAll('script:not([type]), script[type*="javascript"], link[rel=import]');
```

Comment:
@marcomarcianesi, This works fine and I'm ok to carry on with this (specially considering we will throw rendertron away ASAP).

Question to me is: This strips off all javascripts - it already did so, the changes only make sure `application/ld+json` and `application/json` still live on, `text/javascript` are removed anyway. Is this what we want? Because, without the JS, google has no way to do JS crawling..

Which leads me to wonder if this might have something to do with google's ability to crawl us right now... ???